### PR TITLE
Adds the ability to tip medbots over (and they cry too)

### PR DESCRIPTION
**Pull request by swissloaf**
### Intent of your Pull Request

You can now tip medibots over and they will cry out for help. It will get progressively more intense as time passes but they will eventually go right side up if nobody helps them. There are sounds to come with it as well.

>A person is asked to invert three creatures: a Barbie doll, a Furby, and a biological gerbil. [Freedom] Baird's question is simple: "How long can you hold the object upside down before your emotions make you turn it back?"

>People are surprised by how upset they get in this theater of distress. And then they get upset that they are upset. They often try to reassure themselves, saying things like, “Chill, chill, it’s only a toy!” They are experiencing something new: you can feel bad about yourself for how you behave with a computer program. Adults come to the upside-down test knowing two things: the Furby is a machine and they are not torturers. By the end, with a whimpering Furby in tow, they are on new ethical terrain.

Here's a line example (from start to finish w/ sound):

**I require assistance.
Please put me back.
Please, I am scared!
I DON'T LIKE THIS, I NEED HELP!
THIS HURTS, MY PAIN IS REAL!
Is this the end?
Noooo!**

If you don't help it, it will report to you the medbay. Specifically the psych department. Thought that was a neat feature. If you do help it, it will thank and forgive you.

It also has progressive emotes too.

Credit to Ryll-Ryll (https://github.com/tgstation/tgstation/pull/51409) for the PR and Creative Labs for the audio.

#### Changelog

:cl:  
rscadd: Adds the ability to tip medbots over
rscadd: Adds a new emag sound for medbot
rscadd: Medbots can now cry out for help and it will get progressively more intense
soundadd: New medbot sounds
/:cl:

### CODE REVIEW

**Comment by Jumps0** [comment]
you monster

**Comment by Xoxeyos** [comment]
I fully support the endangerment of medbots, no medbot will be left untipped.

**Comment by Nickvr628** [comment]
this is horrifying and amazing

**Comment by athena413** [comment]
speedmerge

**Comment by swissloaf** [comment]
image

**Comment by Xoxeyos** [comment]
Also sound code for "fuck you", which I like.

File: code/__DEFINES/robots.dm [hunk 1]
```diff
@@ -27,6 +27,7 @@
 #define BOT_NAV				15	// computing navigation
 #define BOT_WAIT_FOR_NAV	16	// waiting for nav computation
 #define BOT_NO_ROUTE		17	// no destination beacon found (or no route)
+#define BOT_TIPPED			19	// someone tipped a medibot over ;_;
```
Comment (JamieD1) [inline code/__DEFINES/robots.dm]:
```suggestion
#define BOT_TIPPED			18	// someone tipped a medibot over ;_;
```

File: code/modules/mob/living/simple_animal/bot/medbot.dm [hunk 2]
```diff
@@ -288,11 +385,17 @@
 		soft_reset()
 
 	if(QDELETED(patient))
-		if((emagged || !shut_up) && prob(emagged ? 8 : 1))
-			var/list/messagevoice = list("Radar, put a mask on!" = 'sound/voice/medbot/radar.ogg',"There's always a catch, and I'm the best there is." = 'sound/voice/medbot/catch.ogg',"I knew it, I should've been a plastic surgeon." = 'sound/voice/medbot/surgeon.ogg',"What kind of medbay is this? Everyone's dropping like flies." = 'sound/voice/medbot/flies.ogg',"Delicious!" = 'sound/voice/medbot/delicious.ogg')
-			var/message = pick(messagevoice)
-			speak(message)
-			playsound(src, messagevoice[message], 50)
+
```
Comment (JamieD1) [inline code/modules/mob/living/simple_animal/bot/medbot.dm]:
```suggestion
```

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**